lib/src/rules/omit_local_variable_types.dart

```diff
@@ -8,12 +8,15 @@ import 'package:analyzer/dart/element/type.dart';
 import 'package:linter/src/analyzer.dart';
 import 'package:linter/src/util/dart_type_utilities.dart';
 
-const _desc = r'Omit the types for local variables.';
+const _desc = r'Consider omitting the types for local variables.';
```

Comment:
While the guideline leaves room for choice, the lint rule doesn't. Do we want the description to match the guideline, or what the rule is telling users?